src/main/java/org/team1540/robot2018/Robot.java

```diff
@@ -87,6 +78,7 @@ public void robotInit() {
     driveMode.addObject("Manual Override", true);
     SmartDashboard.putData("[Drivetrain] ***** DRIVE OVERRIDE *****", driveMode);
 
+    // TODO: Move SmartDashboard commands to separate class
```

Comment:
We could maybe split it into separate functions in a static utility class, but since we're only using them once I don't see much benefit over just labeling them nicely

Reply:
Maybe everything (besides tuning values) that's being put to the SmartDashboard could go in one static class?

Reply:
If you want. Or just a static method in `Robot`